test/output/build/data-loaders/_import/test.86a60bc6.js

```diff
@@ -1,3 +1,3 @@
-import {FileAttachment} from "../_observablehq/stdlib.00000003.js";
+import {FileAttachment} from "../_observablehq/stdlib.00000003.js"/* observablehq-file */;
 
-export const test = FileAttachment({"name":"../test.txt","mimeType":"text/plain","path":"../_file/test.f2ca1bb6.txt","lastModified":/* ts */1706742000000,"size":5}, import.meta.url).text();
+export const test = FileAttachment({"name":"../test.txt","mimeType":"text/plain","path":"../_file/test.f2ca1bb6.txt"/* observablehq-file */,"lastModified":/* ts */1706742000000,"size":5, import.meta.url).text();
```

Comment:
You can see the missing `}` here too. This should be the below, I think
```suggestion
export const test = FileAttachment({"name":"../test.txt","mimeType":"text/plain","path":"../_file/test.f2ca1bb6.txt"/* observablehq-file */,"lastModified":/* ts */1706742000000,"size":5}, import.meta.url).text();
```